bundles/mapping/camera-controls-3d/view/CameraControls3d.jsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { withLocale } from 'oskari-ui/util';
+import { MoveMapIcon, RotateMapIcon, UpIcon, DownIcon } from './CameraControls3d/CameraControl3dIcons';
+
+const upDownChangePercent = 10;
+const iconShadow = '1px 1px 2px rgba(0,0,0,0.6)';
+const darkBgColor = 'rgba(20,20,20,0.8)';
+// TODO: Change secondary color reference later when global way available
+const secondaryColor = '#006ce8';
+const mapMoveMethodMove = 'move';
+const mapMoveMethodRotate = 'rotate';
+
+const MapControlsContainer = styled.div`
+    margin-top: 10px;
+    margin-left: 5px;
+    display: flex;
+    flex-wrap: wrap;
+    width: 80px;
+    align-items: center;
+`;
+
+const MapControlContainer = styled.div`
+    border-radius: 50%;
+    width: 32px;
+    height: 32px;
+    cursor: pointer;
+    z-index: 15000;
+    margin-bottom: 5px;
+    margin-left: 5px;
+    box-shadow: ${iconShadow};
+    -moz-box-shadow: ${iconShadow};
+    -webkit-box-shadow: ${iconShadow};
+    -o-box-shadow: ${iconShadow};
```

Comment:
Styled-components promotes "automatic vendor prefixing" as a feature so these can be removed.

Reply:
Done